shared/cryptoutil/cryptoUtil.ts

```diff
@@ -190,3 +196,9 @@ export function hmacSHA(encryptionKey: string, data: string) {
   const resultBytes = hmac.digest().getBytes().toString();
   return resultBytes;
 }
+
+export async function compressData(fileName: string): Promise<string> {
+  const result = await RNZipArchive.zip(backupFilePath, zipFilePath(fileName));
+  await fileStorage.removeItem(getBackupFilePath(fileName));
```

Comment:
This function should strictly compress the contents and not do any removal as the fn name suggests otherwise.

Reply:
Makes sense. Our use case requires us to remove the file hence added this. I will add a wrapper function to make it generic